user/views.py

```diff
@@ -0,0 +1,19 @@
+from django.views.generic.base import TemplateView
+from django.views.generic.base import View
+from django.shortcuts import render
+
+from .forms import UserRegistrationForm
+
+class HomeView(TemplateView):
+	template_name = 'home.html'
+	
+
+class RegisterView(View):
+	def post(self, request, *args, **kwargs):
+		form = UserRegistrationForm(request.POST)
+		if form.is_valid():
+			form.save()
+		return render(request, 'register.html')
+	
+class Register(TemplateView):
```

Comment:
Why do we have this other view, You can use the above view for handling GET request